client/signup/steps/site-information/index.jsx

```diff
@@ -0,0 +1,226 @@
+											</InfoPopover>
+										</FormLabel>
+										<FormTelInput
+											id="phone"
+											name="phone"
+											placeholder={ translate( 'E.g. (555) 555-5555' ) }
+											onChange={ this.handleInputChange }
+											value={ this.state.phone }
+										/>
+									</FormFieldset>
+								</Fragment>
+							) }
+							<div className="site-information__submit-wrapper">
+								<Button primary={ true } type="submit" onClick={ this.handleSubmit }>
+									{ translate( 'Continue' ) }
+								</Button>
+							</div>
+						</Card>
+					</form>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { flowName, positionInFlow, signupProgress, stepName, translate } = this.props;
+
+		const headerText = translate( 'Almost done, just a few more things.' );
+		const subHeaderText = translate( "We'll add this information to your new website." );
+
+		return (
+			<StepWrapper
+				flowName={ flowName }
+				stepName={ stepName }
+				positionInFlow={ positionInFlow }
+				headerText={ headerText }
+				fallbackHeaderText={ headerText }
+				subHeaderText={ subHeaderText }
+				fallbackSubHeaderText={ subHeaderText }
+				signupProgress={ signupProgress }
+				stepContent={ this.renderContent() }
+			/>
+		);
+	}
+}
+
+/**
+ * Returns translated label and placeholder for the name field depending on the site type
+ *
+ * @param   {String} siteType E.g,'business', 'blog'
+ * @returns {Object} label and placeholder object
+ */
+function getSiteNameText( siteType ) {
+	//TODO: After site-type component is merged, fetch segment names from lib/site-type.js
+	switch ( dasherize( siteType ) ) {
+		case 'business':
+			return {
+				label: i18n.translate( 'Business name' ),
+				placeholder: i18n.translate( "E.g. Juliana's Pizza" ),
+			};
+		case 'blog':
+			return {
+				label: i18n.translate( 'Blog name' ),
+				placeholder: i18n.translate( 'E.g. My travel blog ' ),
+			};
+	}
+	return {
+		label: i18n.translate( 'Site name' ),
+		placeholder: i18n.translate( 'E.g. My portfolio' ),
+	};
+}
+
+export default connect(
+	state => {
+		const siteType = getSiteType( state );
+		return {
+			isLoggedIn: isUserLoggedIn( state ),
+			siteInformation: getSiteInformation( state ),
+			siteTitle: getSiteTitle( state ),
+			isBusinessSiteSelected: dasherize( 'business' ) === dasherize( siteType ),
+			siteNameText: getSiteNameText( siteType ),
+		};
+	},
+	( dispatch, ownProps ) => ( {
+		submitStep: ( { name, address, email, phone } ) => {
+			address = trim( address );
+			email = trim( email );
+			phone = trim( phone );
+			dispatch( setSiteTitle( trim( name ) ) );
+			dispatch( setSiteInformation( { address, email, phone } ) );
+			// Create site
+			SignupActions.submitSignupStep(
+				{
+					processingMessage: i18n.translate( 'Collecting your information' ),
+					stepName: ownProps.stepName,
+				},
+				[],
+				{
+					address,
+					email,
+					phone,
+				}
+			);
+			ownProps.goToNextStep( ownProps.flowName );
```

Comment:
I've gone ahead and added the tracks event, since we're doing this in every other step. @michaeldcain do let me know if you think it's not needed here.